login.css

```diff
@@ -57,10 +57,10 @@ ul li ul{
 	padding: 0;
 	position: absolute;
 	top: 58px;
-	left: 130px;
```

Comment:
08/08/2021: I delete the left because I have rearrange the margin in the link so it was correct. That mean I don't need to add left so I could delete this.